src/main/java/ch/uzh/ifi/hase/soprafs22/controller/LobbyController.java

```diff
@@ -135,15 +135,27 @@ public List<LobbyGetDTO> getLobby() {
         return lobbiesGetDTOs;
     }
 
-    @GetMapping("/{apiVersion}/game/match/{lobbyId}")
+    @PostMapping("/{apiVersion}/game/match/{lobbyId}")
     @ResponseStatus(HttpStatus.CREATED)
     @ResponseBody
```

Comment:
Since the method is void, should we remove the annotation @ResponseBody?

Reply:
ah yes, I thought that was necessary. I will remove it